storage/remote/intern_test.go

```diff
@@ -74,17 +73,21 @@ func TestIntern_MultiRef_Concurrent(t *testing.T) {
 	interner.intern(testString)
 	interned, ok := interner.pool[testString]
 	require.True(t, ok)
-	require.Equalf(t, int64(1), interned.refs.Load(), "expected refs to be 1 but it was %d", interned.refs.Load())
-
-	go interner.release(testString)
-
-	interner.intern(testString)
-
-	time.Sleep(time.Millisecond)
+	require.Equalf(t, int64(1), interned.refs.Load(), "wrong interned refs count")
+
+	for i := 0; i < 1000; i++ {
+		released := make(chan struct{})
+		go func() {
+			interner.release(testString)
+			close(released)
+		}()
+		interner.intern(testString)
+		<-released
+	}
 
 	interner.mtx.RLock()
 	interned, ok = interner.pool[testString]
 	interner.mtx.RUnlock()
 	require.True(t, ok)
-	require.Equalf(t, int64(1), interned.refs.Load(), "expected refs to be 1 but it was %d", interned.refs.Load())
+	require.Equalf(t, int64(1), interned.refs.Load(), "wrong interned refs count")
```

Comment:
Note for reviewers: `require.Equal` will already say "expected X but got Y", there's no need to say that, we just need to explain that difference.

Why did I change this? Because we were doing `interned.refs.Load()` twice, so the numbers could be different between the one passed to the message and to the expectation.